dashboard/app/controllers/test_controller.rb

```
@@ -214,6 +251,12 @@ def destroy_script
     head :ok
   end
 
+  def destroy_course
+    course = UnitGroup.find_by!(name: params[:course_name])
+    course.destroy
```

this will silently ignore a failure in deleting the course. how about `destroy!` instead?